recipes-multimedia/gstreamer/gstreamer1.0_1.18.0.bb

```diff
@@ -6,41 +6,38 @@ BUGTRACKER = "https://bugzilla.gnome.org/enter_bug.cgi?product=Gstreamer"
 SECTION = "multimedia"
 LICENSE = "LGPLv2+"
 
-DEPENDS = "glib-2.0 glib-2.0-native libcap libxml2 bison-native flex-native"
+DEPENDS = "glib-2.0 glib-2.0-native libxml2 bison-native flex-native"
 
-inherit meson pkgconfig gettext upstream-version-is-even gobject-introspection gtk-doc
+inherit meson pkgconfig gettext upstream-version-is-even gobject-introspection
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=6762ed442b3822387a51c92d928ead0d \
                     file://gst/gst.h;beginline=1;endline=21;md5=e059138481205ee2c6fc1c079c016d0d"
 
 S = "${WORKDIR}/gstreamer-${PV}"
 
-SRC_URI = " \
-    https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-${PV}.tar.xz \
-    file://0001-gst-gstpluginloader.c-when-env-var-is-set-do-not-fal.patch \
-    file://0002-meson-build-gir-even-when-cross-compiling-if-introsp.patch \
-    file://0003-meson-Add-valgrind-feature.patch \
-    file://0004-meson-Add-option-for-installed-tests.patch \
-    file://capfix.patch \
-"
-SRC_URI[md5sum] = "0e661ed5bdf1d8996e430228d022628e"
-SRC_URI[sha256sum] = "e3f044246783fd685439647373fa13ba14f7ab0b346eadd06437092f8419e94e"
+SRC_URI = "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-${PV}.tar.xz \
+           file://0001-gst-gstpluginloader.c-when-env-var-is-set-do-not-fal.patch \
+           file://0002-Remove-unused-valgrind-detection.patch \
+           file://0003-meson-Add-option-for-installed-tests.patch \
+           "
+SRC_URI[md5sum] = "25d683630ff8b82b2c878aa3edd3c491"
+SRC_URI[sha256sum] = "0ff09245b06c0aeb5d9a156edcab088a7e8213a0bf9c84a1ff0318f9c00c7805"
 
 PACKAGECONFIG ??= "${@bb.utils.contains('PTEST_ENABLED', '1', 'tests', '', d)} \
                    check \
                    debug \
-                   tools"
+                   tools \
+                   capabilities"
```

Comment:
The capabilities addition should be added as another patch as it is changing the current build settings.

Reply:
I have mention this change on the commit message
```
    * capfix.patch
      Removed as the new PACKAGECONFIG capabilities can be used instead to achieve the same goals.
      Fix reproducibility issue around libcap.
```
But after review this again it has a different behavior in this patch set

On the 1.16 it uses 'setuid-root'` if the 'setcap' is not provided in PACKAGECONFIG
I will need to do the same behavior and remove the new 'capabilities' from default PACKAGECONFIG

Reply:
I can remove the PACKAGECONFIG `capabilities` and use the  `setcap`  to do the same.
With this changes I only drop the patch `capfix.patch` and adjusts the `setcap`.

```patch
-PACKAGECONFIG[setcap] = "-Dsetcap=enabled,-Dsetcap=disabled,libcap libcap-native"
+PACKAGECONFIG[setcap] = ",,libcap libcap-native"
```

Reply:
I also remove `libcap` from `DEPENDS` because it is only needed to set the capabilities.
Now it as the same behavior of the version 1.16.